test/sources/source.t/run.t

```diff
@@ -135,21 +135,18 @@ Ids generated in the source code:
   id="L18"
   id="L19"
   id="L20"
-  id="def-0"
-  id="def-1"
-  id="x_268"
-  id="def-2"
-  id="y_269"
-  id="def-3"
-  id="z_270"
-  id="a_272"
-  id="def-5"
-  id="def-6"
-  id="def-7"
-  id="def-8"
-  id="def-9"
-  id="def-10"
-  id="def-11"
-  id="def-12"
-  id="def-14"
-  id="def-15"
+  id="type-t"
+  id="value-x"
+  id="value-y"
+  id="value-z"
+  id="local_a_47"
+  id="module-A"
+  id="module-B"
+  id="module_type-T"
+  id="module_type-U"
+  id="type-ext"
+  id="def_10"
+  id="def_11"
```

Comment:
The `def_%d` identifiers are for `Uids` whose location was not found during the Typedtree traverse.